File: SeriesGuide/src/main/java/com/battlelancer/seriesguide/extensions/GooglePlayExtensionReceiver.java
package com.battlelancer.seriesguide.extensions;

import com.battlelancer.seriesguide.SgApp;
import com.battlelancer.seriesguide.api.SeriesGuideExtension;
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver;

public class GooglePlayExtensionReceiver extends SeriesGuideExtensionReceiver {
    @Override
    protected int getJobId() {
        return SgApp.JOB_ID_EXTENSION_GOOGLE_PLAY;
    }

    @Override
    protected Class<? extends SeriesGuideExtension> getExtensionClass() {
        return GooglePlayExtension.class;
    }
}

File: app/src/main/java/com/battlelancer/seriesguide/extensions/GooglePlayExtensionReceiver.kt
package com.battlelancer.seriesguide.extensions

import com.battlelancer.seriesguide.SgApp
import com.battlelancer.seriesguide.api.SeriesGuideExtension
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver

class GooglePlayExtensionReceiver : SeriesGuideExtensionReceiver() {
    override fun getJobId(): Int = SgApp.JOB_ID_EXTENSION_GOOGLE_PLAY

    override fun getExtensionClass(): Class<out SeriesGuideExtension> =
        GooglePlayExtension::class.java
}